### ISSUE TITLE
local variable 'slot_date_second' referenced before assignment

### ISSUE THREAD

**Issue by gitolicious** [rbignon/doctoshotgun#75]
I was running the script as always (default settings in Germany) when a Pfizer vaccination showed up. Unfortunately it crashed with `local variable 'slot_date_second' referenced before assignment`.

```
Center Gemeinschaftspraxis Dr. med. M. Hedayatian und Maria Engelbrecht:
– Gemeinschaftspraxis Dr. med. M. Hedayatian und Maria Engelbrecht...
  Vaccine Pfizer...
An unexpected exception of type UnboundLocalError occurred. Arguments:
("local variable 'slot_date_second' referenced before assignment",)
```

The assignment went wrong here:

https://github.com/rbignon/doctoshotgun/blob/10cfc0ea70c694eeaaa964eda4ab62107f497c95/doctoshotgun.py#L325-L337

The actual result was of type `str` (`"2021-06-22T15:40:00.000+02:00"`). It works fine for other runs...
So there are multi-shot vaccinations where the results actually are of type `str`. This is where it happened: https://www.doctolib.de/gemeinschaftspraxis/neuss/gemeinschaftspraxis-dr-med-m-hedayatian-und-maria-engelbrecht

I guess the doctor didn't configure Doctolib correctly, still the script should not crash on this.

**Comment by gitolicious** [comment]
Problem was that second-shot-only appointments got selected by `find_motive` which did not have a `slot_date_second`. This is avoided with #78.

**Comment by gitolicious** [comment]
Fixed by #97
